vendor/github.com/minio/minio-go/pkg/s3signer/request-signature-v2.go

```diff
@@ -253,17 +251,27 @@ func writeCanonicalizedHeaders(buf *bytes.Buffer, req http.Request) {
 	}
 }
 
-// The following list is already sorted and should always be, otherwise we could
-// have signature-related issues
+// AWS S3 Signature V2 calculation rule is give here:
```

Comment:
typo in comment:
`// AWS S3 Signature V2 calculation rule is give here:`
=???=> give**n**